Source/Classes/AnimatedImages/PINAnimatedImageView.m

```diff
@@ -391,19 +398,37 @@ - (void)displayLinkFired:(PINDisplayLink *)displayLink
     }
 }
 
+- (void)resetDurationsWithAnimatedImage:(PINCachedAnimatedImage *)animatedImage
```

Comment:
Should this method just be a part of initializeAnimatedImage?

Reply:
🤔 Yes, I'll move the caller in `initializeAnimiatedImage` method.